lib/teamsnap/version.rb

```diff
@@ -1,3 +1,3 @@
 module TeamSnap
-  VERSION = "3.0.1"
+  VERSION = "3.1.0"
```

Comment:
```suggestion
  VERSION = "3.2.0"
```

Reply:
actually i didn't change the module, so im just going to revert this change all together